e2e-pw/src/oss/poms/panels/embeddings-panel.ts

```diff
@@ -0,0 +1,75 @@
+import { Locator, Page, expect } from "src/oss/fixtures";
+import { EventUtils } from "src/shared/event-utils";
+import { SelectorPom } from "../selector";
+import { PanelPom } from "./panel";
+
+export class EmbeddingsPom {
+  readonly locator: Locator;
+  readonly plotContainer: Locator;
+  readonly selector: SelectorPom;
+  readonly colorbySelector: SelectorPom;
+  readonly asserter: EmebddingsAsserter;
+  readonly pp: Locator;
```

Comment:
What is `pp`? (I see that it's `page` but as a user of this POM, I wouldn't know)